src/imitation/scripts/config/train_adversarial.py

```diff
@@ -121,19 +121,13 @@ def airl():
   )
 
 
-# Standard Gym environment named configs
+# Classic RL Gym environment named configs
 
 @train_ex.named_config
 def acrobot():
   env_name = "Acrobot-v1"
   rollout_hint = "acrobot"
-
-
-@train_ex.named_config
-def ant():
-  env_name = "Ant-v2"
-  rollout_hint = "ant"
-  locals().update(**ant_shared_locals)
+  locals().update(**mujoco_shared_locals)
```

Comment:
Acrobot is not MuJoCo. Maybe you want to use these params anyway? But at least add a comment explaining why.

Reply:
Ah, my mistake there